zerver/lib/upload.py

```diff
@@ -165,6 +168,23 @@ def upload_image_to_s3(
 
     key.set_contents_from_string(force_str(contents), headers=headers)
 
+def get_total_qouta(user):
```

Comment:
maybe call this `get_total_uploads_size_for_user`, to make it more clear that this is the used quote, not the total quota; and it's for a user, not a realm?